pkg/volume/volume_info_v1.go

```diff
@@ -0,0 +1,38 @@
+package volume
+
+import metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+type VolumeInfosV1 struct {
+	Infos []VolumeInfoV1 `json:"infos"`
+
+	// VolumeInfo structure's version information.
+	Version string `json:"version"`
+}
+
+type VolumeInfoV1 struct {
+	// The PVC's name. The format should be <namespace-name>/<PVC-name>
+	PVCName string `json:"pvcName"`
+
+	// The PV name.
+	PVName string `json:"pvName"`
+
+	// The way the volume data is backed up. The valid value includes `VeleroNativeSnapshot`, `PodVolumeBackup` and `CSISnapshot`.
+	BackupMethod string `json:"backupMethod,omitempty"`
+
+	// Whether the volume's snapshot data is moved to specified storage.
+	SnapshotDataMovement bool `json:"snapshotDataMovement"`
```

Comment:
Considering that the type is boolean, wouldn't it be better to call the field "SnapshotDataMoved"?

Reply:
Thanks. Modified.